src/routes/index.js

```
@@ -11,6 +11,7 @@
 const country = require('~/routes/countries')
 const uploadRoutes = require('~/routes/uploadPhoto')
 const subject = require('~/routes/subject')
+const languageRoutes = require('./routes/languageRoutes');
```

ESLint

[error] Extra semicolon.

(semi)